com.rioterm.Rio.yml

```diff
@@ -0,0 +1,54 @@
+---
+app-id: com.rioterm.Rio
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-config/rio
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
Without this flag, it can't open the shell and immediately crashes

Reply:
this need an exception

Reply:
An exception to what?

Reply:
https://docs.flathub.org/docs/for-app-authors/linter#exceptions

Reply:
and

https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access

Reply:
Exception added :)